package.json

```diff
@@ -5,6 +5,7 @@
     "dev": "next",
     "build": "next build",
     "lint": "eslint '**/*.js' '**/*.jsx'",
+    "lint-fix": "eslint --fix '**/*.js' '**/*.jsx'",
     "start": "next start",
```

Comment:
just a little extra helper which will fix the lint errors for you (and stage the diff to review and edit before commiting it)